packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -255,7 +255,12 @@ class VMService implements vm_service.VmService {
       final Map<String, Object> versionJson = version.toJson();
       versionJson['frameworkRevisionShort'] = version.frameworkRevisionShort;
       versionJson['engineRevisionShort'] = version.engineRevisionShort;
-      return versionJson;
+      return <String, dynamic>{
+        'result': <String, Object>{
+          'type': 'Success',
```

Comment:
This looks correct (comparing to the `compileExpression` implementation).

cc @bkonyi - it looks like each client registering a vm service extension needs to be very careful about the response it creates